maven/impl-maven-embedded/src/main/java/org/jboss/shrinkwrap/resolver/impl/maven/embedded/pom/equipped/ConfigurationStageImpl.java

```diff
@@ -242,4 +243,16 @@ public ConfigurationDistributionStage setResumeFrom(String resumeFrom) {
         getInvocationRequest().setResumeFrom(resumeFrom);
         return this;
     }
+
+    @Override
+    public ConfigurationDistributionStage setAlternatePomFile(String pomFile) {
+        if(pomFile != null) {
```

Comment:
If you do nothing when the `null` is specified, then it should be mentioned in the javadoc